language-docker.cabal

```diff
@@ -0,0 +1,90 @@
+cabal-version: 1.12
+
+-- This file has been generated from package.yaml by hpack version 0.31.2.
```

Comment:
Do you want to include this file when it is autogenerated?

Reply:
Yeah, the new recommendations from stack is to include it. It seems to avoid errors in versioning and makes life of people not using stack easier